CONTRIBUTING.md

```diff
@@ -73,31 +110,34 @@ Snapcraft.
 
 All significant work in Snapcraft should be tied to an existing issue or ticket.
 
+Once you find an issue that you’d like to work on, it’s best to reach out so that a
+Snapcraft maintainer can assign the ticket to you before starting. This process varies
+depending on the issue’s complexity and scope.
+
+#### Minor changes
+
 If you'd like to add a small feature or fix, check the project's GitHub issues to see if
 others have reported it. If they have, look into the current status of the topic. If no
 one else is working on it, add a comment stating that you'd like to take it on, and the
 Starcraft team will assign it to you.
 
-If there's a large feature or fix you'd like to work on, contact the team and the
-community on the [Snapcraft Matrix channel](https://matrix.to/#/#snapcraft:ubuntu.com).
-It's possible that work on it has been started, or that it fits into an existing plan.
-Often, you will save time and effort by checking for prior work.
+If you don’t find a related issue, [open
+one](https://github.com/canonical/snapcraft/issues/new/choose) and indicate that you’re
+interested in taking it on. When creating the issue, be sure to add any relevant labels.
+A Snapcraft maintainer will then review the issue and assign it to you.
 
-If you're ever in doubt about developments in the project, ask!
+#### Major changes
```

Comment:
Would this be a good section to say something like this? I write this out somewhat often in bug reports and on matrix, so it would be nice to write it down:

> The [forum](https://forum.snapcraft.io/) is the best place to discuss major changes that require coordination and development with the Snap Store or with snapd. The forum serves as a central point for all teams in the snap ecosystem, whereas issues in the Snapcraft repo are best suited for changes that only concern Snapcraft.

Reply:
Looks great, much appreciated!